pkg/kms/gcp/gcp.go

```diff
@@ -39,12 +39,13 @@ type KMS struct {
 	locationID    string
 	keyRing       string
 	key           string
+	version       string
 }
 
 var (
 	ErrKMSReference = errors.New("kms specification should be in the format gcpkms://projects/[PROJECT_ID]/locations/[LOCATION]/keyRings/[KEY_RING]/cryptoKeys/[KEY]")
```

Comment:
do you need to update this string to note the optional versions tag?

Reply:
Good catch.